MAINTAINERS.yml

```diff
@@ -474,14 +496,52 @@ Bluetooth Classic:
     - jhedberg
     - sjanc
   files:
+    - subsys/bluetooth/common/
     - subsys/bluetooth/host/classic/
+    - subsys/bluetooth/shell/bredr.c
     - include/zephyr/bluetooth/classic/
   labels:
     - "area: Bluetooth Classic"
     - "area: Bluetooth"
   tests:
     - bluetooth
 
+Bluetooth ISO:
+  status: maintained
+  maintainers:
+    - Thalley
+  collaborators:
+    - jhedberg
+  files:
+    - include/zephyr/bluetooth/iso.h
+    - doc/connectivity/bluetooth/api/shell/iso.rst
+    - samples/bluetooth/*_iso/
+    - samples/bluetooth/iso_*/
+    - subsys/bluetooth/shell/iso.c
+    - subsys/bluetooth/Kconfig.iso
+  labels:
+    - "area: Bluetooth ISO"
+    - "area: Bluetooth"
+  tests:
+    - bluetooth
+
+Bluetooth Qualification:
+  status: maintained
+  maintainers:
+    - sjanc
+  collaborators:
+    - Thalley
+    - jhedberg
```

Comment:
If anyone else wants to be added to these we can either do it later, or just me know in this PR